doc/version_history.rst

```diff
@@ -2,6 +2,16 @@
 Version History
 ===============
 
+v0.11.1
```

Comment:
I think the changes made here are not backward compatible so, even though you are making bug fixes this should be a minor release (e.g.; v0.12.0).

Reply:
Done in the commit: [d7beec9](https://github.com/lsst-ts/ts_m2com/pull/38/commits/d7beec9180505af36e222c539032fea7eadfb7e5). Thanks!